foundation-ui/mount/MountService.js

```diff
@@ -0,0 +1,94 @@
+import {EventEmitter} from 'events';
+import {CHANGE} from './MountEvent';
+
+/**
+ * List of component descriptors
+ *
+ * @type {Array.<{component:function, priority:Number, type:String}>}
+ */
+const components = [];
```

Comment:
I removed that data structure as discussed in https://github.com/dcos/dcos-ui/pull/1130#issuecomment-259173317 to simplify the current implementation as it made the refactoring harder without providing a huge performance gain. I'll reintroduce the optimization once we've landed the packages and see need.